lib/spack/spack/cmd/env.py

```diff
@@ -834,14 +864,14 @@ def env_depfile_setup_parser(subparser):
         "--generator",
         default="make",
         choices=("make",),
-        help="specify the depfile type\n\ncurrently only make is supported",
+        help="specify the depfile type (only support `make`)",
```

Comment:
For some reason "only support make" sounds intentional and not incidental here.

```suggestion
        help="specify the depfile type (only supports `make`)",
```